File: Command/Ruby/Image.rb
class Image
    def initialize(imagePosition, imageSize)
        @imagePosition = imagePosition
        @imageSize = imageSize.to_f
    end

    def rotate(degree)
        @imagePosition = (@imagePosition + degree).abs % 360
    end

    def resize(percentage)
        @imageSize *= (percentage.to_f / 100).to_f
    end

    def to_s
        "Image Position : #{@imagePosition}\nImage Size     : #{@imageSize}"
    end
end

File: Command/Javascript/Image.js
class Image {
  constructor(imagePosition, imageSize) {
    this.imagePosition = imagePosition;
    this.imageSize = imageSize;
  }

  rotate(degree) {
    this.imagePosition = Math.abs(this.imagePosition + degree) % 360;
  }

  resize(percentage) {
    this.imageSize *= (percentage / 100);
  }

  inspect() {
    console.log(`
      Image position: ${this.imagePosition}
      Image size    : ${this.imageSize}
    `);
  }
}

export default Image;